iamf/cli/proto/codec_config.proto

```diff
@@ -41,6 +41,9 @@ message OpusEncoderMetadata {
   optional OpusApplicationFlag application = 2;
   optional bool use_float_api = 3 [default = true];
   optional float coupling_rate_adjustment = 4 [default = 1.0];
+  optional bool use_substream_rate_override = 5 [default = false];
```

Comment:
I think the proto  may be better as 

```
map<uint32, int32> substream_id_to_bitrate_override = 5;
```

In textproto it would look like.

```
# Substream ID 123 has a bitrate of 1000.
substream_id_to_bitrate_override {
key: 123
value: 1000 
}
```

This would be more reasonable if large substream IDs are used - which are generally supported elsewhere in the encoder.

I think in this case it makes sense to check presence of the substream in the map, and there does not need to be a `use_substream_rate_override`. When the substream is not present that implies it is not being overrided - using the "default" calculation would be reasonable.

Reply:
Thanks for the help, was considering it but was not sure how maps worked here.